packages/itwinui-css/src/footer/footer.scss

```diff
@@ -32,12 +31,27 @@
   color: var(--iui-color-text-muted);
 
   > a {
-    @include anchor.iui-anchor;
-
     &,
     &:hover {
       color: var(--iui-color-text-muted);
     }
+
+    &:not(:hover) {
```

Comment:
Is this the same as `&`? Maybe you did this for some specificity conflict?

Reply:
`&` and `&:not(:hover)` are not the same. `&` is a rule that is always applied, while `&:not(:hover)` only applies it's styles when the element is not being hovered over. That change would create a conflict that does resolve correctly in this case, since the `:hover` style from `iui-anchor` has a higher specificity. However, I do not think it is wise to rely on the `:hover` rule winning the specificity battle for the footer links to be styled correctly.

Reply:
> `&` and `&:not(:hover)` are not the same.

True, but I meant more like selecting something when it's not hovered (`&:not(:hover)`) would likely be the same as selecting that something directly without any pseudo classes (`&`), unless we *really* want to select it only when it's not hovered.

Upon rethinking it, are all those `text-decoration` styles required here ([code](https://github.com/iTwin/iTwinUI/pull/1915/files#diff-1cd601586d11db817d96d51d666331f1ea862f34ce736612b6a16204ed128583R39-R54))? Because I think it they all also from the anchor?

Reply:
> > `&` and `&:not(:hover)` are not the same.
> 
> True, but I meant more like selecting something when it's not hovered (`&:not(:hover)`) would likely be the same as selecting that something directly without any pseudo classes (`&`), unless we _really_ want to select it only when it's not hovered.


`&:not(:hover)` was suggested by me. It is overriding the default user-agent styles of `<a>` element (which comes with an underline)

> Upon rethinking it, are all those `text-decoration` styles required here ([code](https://github.com/iTwin/iTwinUI/pull/1915/files#diff-1cd601586d11db817d96d51d666331f1ea862f34ce736612b6a16204ed128583R39-R54))? Because I think it they all also from the anchor?

`Anchor` may not always be used, like in the Custom Content Footer story

Reply:
These styles are specifically being included because the `Footer` component allows for custom content. Removing these styles would break the styling for any custom content that does not use the `iui-anchor` class, which would introduce a breaking change to the react components for any users using custom content. As a result, it was decided to keep the styles here. Relevant conversation: [here](https://github.com/iTwin/iTwinUI/pull/1915#discussion_r1525481818)

Reply:
Ohh, I see, I now understand why we need those styles.

In high contrast, the underline even shows up on hover. Is this expected? ([demo](https://itwin.github.io/iTwinUI/1915/react/?arg-high-contrast=true&story=footer--custom-content&theme=dark))

https://github.com/iTwin/iTwinUI/assets/45748283/adeda38d-5ae2-438f-9127-42ccd174b39d

Reply:
I think that's ok. In a future PR, I would like to make all anchors have underlines on hover, even in high-contrast theme.